spring-cloud-gcp-vision/src/main/java/com/google/cloud/spring/vision/CloudVisionTemplate.java

```diff
@@ -78,6 +94,31 @@ public String extractTextFromImage(Resource imageResource, ImageContext imageCon
     return result;
   }
 
+  /**
+   * Extract the text out of a file and return the result as a String.
+   *
+   * @param fileResource the file one wishes to analyze
+   * @param mimeType the mime type of the fileResource
```

Comment:
Either list the supported mime types or link to docs that do.

Reply:
added the supported types to the doc